Src/CSharpier/SyntaxPrinter/SyntaxNodePrinters/WhenClause.cs

```diff
@@ -7,9 +8,14 @@ public static class WhenClause
     {
         public static Doc Print(WhenClauseSyntax node)
         {
-            return Doc.Concat(
-                Token.PrintWithSuffix(node.WhenKeyword, " "),
-                Node.Print(node.Condition)
+            var groupId = Guid.NewGuid().ToString();
+            return Doc.GroupWithId(
```

Comment:
Is there any particular reason you're making this a `GroupWithId` instead of just a `Group`? 
It doesn't look like you're doing anything with the id.

Reply:
Ah - I had thought I needed to use `IndentIfBreak`. I'll change it to `Group`.